compiler/src/test/kotlin/dev/zacsweers/metro/compiler/TestUtils.kt

```diff
@@ -422,7 +422,7 @@ fun Class<*>.createGraphWithNoArgs(): Any {
 fun Class<*>.createGraphViaFactory(vararg args: Any): Any {
   val factoryInstance = invokeGraphFactory()
   return factoryInstance.javaClass.declaredMethods
-    .single { it.name == Symbols.StringNames.CREATE }
+    .first { it.name == Symbols.StringNames.CREATE }
```

Comment:
Found out that if the graph creation function lives in a super type, there will be multiple "create" functions in the declared methods. I changed `single` to `first` to unblock the test for now

Reply:
yep, I'll do that